h2o-py/h2o/frame.py

```diff
@@ -1221,35 +1221,36 @@ def structure(self):
                 print("num {}".format(" ".join(it[0] if it else "nan" for it in h2o.as_list(self[:10, i], False)[1:])))
 
 
-    def as_data_frame(self, use_pandas=True, header=True):
+    def as_data_frame(self, use_pandas=True, header=True, strip_nas=True):
         """
         Obtain the dataset as a python-local object.
 
         :param bool use_pandas: If True (default) then return the H2OFrame as a pandas DataFrame (requires that the
             ``pandas`` library was installed). If False, then return the contents of the H2OFrame as plain nested
             list, in a row-wise order.
         :param bool header: If True (default), then column names will be appended as the first row in list
+        :param strip_nas: If True (default), then the NaNs are removed from pandas DataFrame (in case of list of lists of strings, empty lists contain 'NaN').
 
         :returns: A python object (a list of lists of strings, each list is a row, if use_pandas=False, otherwise
             a pandas DataFrame) containing this H2OFrame instance's data.
         """
         if can_use_pandas() and use_pandas:
             import pandas
-            return pandas.read_csv(StringIO(self.get_frame_data()), low_memory=False)
-        frame = [row for row in csv.reader(StringIO(self.get_frame_data()))]
+            return pandas.read_csv(StringIO(self.get_frame_data(strip_nas)), low_memory=False)
+        frame = [row for row in csv.reader(StringIO(self.get_frame_data(strip_nas)))]
         if not header:
             frame.pop(0)
         return frame
 
 
-    def get_frame_data(self):
+    def get_frame_data(self, strip_nas=True):
```

Comment:
I believe that it always has been stripping nas. This PR introduces option not to strip them. Therefore the default True.